service/src/main/resources/api/service_openapi.yaml

```diff
@@ -312,7 +312,26 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/ServerError'
-  ## END OF DEPRECATED PATHS ##
+
+  /api/workspaces/v1/{workspaceId}/resources/referenced/{resourceId}:
```

Comment:
I'm confused by this path. I thought it would have `validate` or something in it. Where do I go if I just want to get a referenced resource?

Reply:
I don't think you can just fetch a single resource without knowing its type beforehand. You can enumerate all the resources, or call the type-specific endpoint.

Reply:
Oh, interesting. Good point.

Reply:
^ yep, this was part of the big discussion around generic vs. resource-specific return types from the API. That said, we might consider supporting a generic GET endpoint in the future, so it's probably prudent to add `/access` here. Does the new URL seem reasonable?